### ISSUE TITLE
Strapi listens to `./`

### ISSUE THREAD

**Issue by DenuxPlays** [strapi/strapi#19019]
### Required System information

- Node.js version: 18.18.2
- NPM version: 9.8.1
- Strapi version: 4.15.5
- Database: MariaDb 10.6
- Operating system: Debian-12 
- Is your project Javascript or Typescript: Typescript

#### Some information

We have two strapis (both in docker) running in parallel.
Both of them have a seperate dist folder (dist and dist_test).
We mount the whole root directory into the docker container.

#### Acutal bug

When (e.g. strapi_test) starts and writes things to their dist (dist_test) folder the other strapi instance also rebuilds and then they need a few minutes until both are fully started.
In general strapi rebuilds on any file change even if the filde doesn't affect strapi.
In short: its annoying please fix!

### Steps to reproduce the behavior

1. Go to your root director
2. Create a file (that doesn't has anything todo with strapi.)
3. Watch strapi restart...
4. Do this for a week because you are documenting your stuff, etc.
5. be annoyed

### Expected behavior

Don't rebuild on any file change...

**Comment by DenuxPlays** [comment]
![image](https://github.com/strapi/strapi/assets/83671398/8b819ebd-04aa-4007-b4fd-a78cc13708d5)
Also this happens quit a lot:
Spoiler: dist_test isn't the dist folder for  this instance.... but still tried to compile it...

**Comment by gu-stav** [comment]
Closing as duplicate of https://github.com/strapi/strapi/issues/18196
